readme/api/tutorials/cm6_plugin.md

```diff
@@ -0,0 +1,634 @@
+---
+sidebar_position: 2
+---
+# Creating a CodeMirror 6 plugin
```

Comment:
We should generally not use the term "CodeMirror" which may not mean much to many users, and instead prefer "Markdown editor". Where needed we can mention that the Markdown editor is based on CM6, in particular for development purposes, but it shouldn't be in the title or prominent places.

Even in commit, which end up in release notes, we should use user-facing terms - "Beta Markdown editor", "Legacy Markdown editor", etc.